index.js

```diff
@@ -128,6 +129,7 @@ class NetworkResource extends Nanoresource {
     function onlisten () {
       self.discovery = discovery({ socket: self.utp })
       self._onopen()
+      self._onbind()
```

Comment:
Oh maybe I’m confused. Whats the diff betwwn onbind and onopen?

Reply:
Ahaha it appears there’s none - it’s just documented with the bind option instead the open option - 

We can either update doc or code.. probably doc if anyone’s relying on open?

Or document an alias?

Reply:
Lol, let’s document

Reply:
And rename it to bind